javascript/javascript1/week2/jsweek2.js

```diff
@@ -0,0 +1,103 @@
+// Flight booking fullname function
+function getFullname(firstname, surname, useFormalName) {
+    if (useFormalName) {
+        return "Lord " + firstname + " " + surname;
+    } else {
+        return firstname + " " + surname;
+    }
+}
+const fullname1 = getFullname("Benjamin", "Hughes", true);
+const fullname2 = getFullname("Benjamin", "Hughes", false);
+
+console.log(fullname1);
+console.log(fullname2);
+
+
+
+// Event application
+const wDays7 = [
+    "Thursday",
+    "Friday",
+    "Saturday",
+    "Sunday",
+    "Monday",
+    "Tuesday",
+    "Wednesday",
+];
+function getEventWeekday(daysOfEvent) {
+    const fullD = new Date();
+    const manyDaysFromToday = new Date(
+        fullD.getDate(),
+        fullD.getMonth() + daysOfEvent
+    );
+    return wDays7[manyDaysFromToday.getDay()];
+}
+// With todays weekday a tuesday
+console.log(getEventWeekday(9)); // Logs out "Thursday"
+
+// With todays weekday a Friday
+console.log(getEventWeekday(5)); // Logs out "Sunday"
+
+
+
+// Weather wear
+function weatherWear(temperature) {
+    if (temperature < 30) {
+        temperature = "Jaket in cold weather";
+    } else {
+        temperature = "T-shirt in hot weather";
+    }
+    return `Hi you should wearing a ${temperature}`;
+}
+console.log(weatherWear(10));
+
+
+
+// Student manager
+const class07Students = [];
+function addStudentToClass(studentName) {
+    if (class07Students.length > 6) {
+        console.log("Cannont add more students to class 07");
+    } else if (class07Students.includes(studentName)) {
+        console.log("studen" + studentName + "is already in the class 07.");
+    } else if (studentName === "Queen") {
+        class07Students.push(studentName);
+    } else if (studentName === "") {
+        console.log("You cannot add an studen without name,please add name.");
+    } else {
+        class07Students.push(studentName);
+    }
+}
+function getNumberOfStudents() {
+    return class07Students.length;
+}
+addStudentToClass("");
+addStudentToClass("ali");
+addStudentToClass("ali");
+addStudentToClass("Alex");
+addStudentToClass("Nine");
+addStudentToClass("Susan");
+addStudentToClass("Marco");
+addStudentToClass("Vital");
+addStudentToClass("Queen");
+console.log(getNumberOfStudents());
```

Comment:
Really good implementation. You clearly considered the conditions in the if else statement thoroughly. 👌

**A few things can be modified slightly for the solution to be on point** 
- If you add a studen when the size of the class is exactly 6, then you are still allowed to. How can you change that in your current implementation? 
- What happens if the Queen is added when the size of the class >=6? According to the assignment description, she should be allowed to enter the class, no matter how many students are currently enrolled. How can you update this in the current solution? 💡

There are some spaces missing in the string concatenation here and there. Maybe you could play around with template literals in this implementation, as you have previously done so well. 💡💪🏼